.github/workflows/robot-command-dispatcher.yaml

```diff
@@ -0,0 +1,56 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+name: Kroxylicious Robot Command Dispatcher
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  process-command:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }} && contains(github.event.comment.body, '@kroxylicious-robot')
+    steps:
+      - name: 'Check team membership'
+        if: ${{ github.repository == 'kroxylicious/kroxylicious' }}
+        uses: tspascoal/get-user-teams-membership@v2
+        id: team-membership
+        with:
+          username: ${{ github.actor }}
+          team: admins
+          GITHUB_TOKEN: ${{ secrets.KROXYLICIOUS_RELEASE_TOKEN }}
+      - name: 'Stop workflow if user is not a team member'
+        if: ${{ github.repository == 'kroxylicious/kroxylicious' && steps.team-membership.outputs.isTeamMember == 'false' }}
+        run: |
+          gh issue comment  ${{ github.event.issue.number }} --body "${{ github.actor }} is not entitled to use @kroxylicious-robot"
+          exit -1
+      - name: 'Extract command'
+        run: |
+          echo "COMMAND=$(echo "${{ github.event.comment.body }}" | awk '/^@kroxylicious-robot/ {print $2}')" >> $GITHUB_ENV
+          echo "Command is ${COMMAND}"
```

Comment:
When I saw `robot-command-dispatcher` I was hoping for some better pre-built options. Happy this does what we need but the support seems a bit lacking.

Reply:
I was trying to act of your feedback you left on the issue.  You expressed a wish for something generic.   In the PR, I'm trying to keep the dispatch workflow command neutral and leave it up to the sub flows to do the work.

Reply:
Sorry I meant I was expecting something better in terms of bot support from GitHub not this PR! 

I think what you've got is absolutely fine.

Reply:
It is my first time working with actions triggered by comments, so it could well be there are much better techniques.